### PR TITLE
allow setting pool name (rebase)

### PR DESCRIPTION
**Pull request by frewsxcv**

### CODE REVIEW

**Comment by frewsxcv** [comment]
@siddontang @BusyJay This look alright?

File: src/lib.rs [hunk 1]
```diff
@@ -105,6 +112,15 @@ impl ThreadPool {
...
     /// This function will panic if `threads` is 0.
     pub fn new(threads: usize) -> ThreadPool {
+        ThreadPool::new_pool(None, threads)
+    }
+
+    pub fn new_with_name(name: String, threads: usize) -> ThreadPool {
+        ThreadPool::new_pool(Some(name), threads)
+    }
+
+    #[inline]
```
Comment (frewsxcv) [inline src/lib.rs]: Note that I added this `#[inline]`

**Comment by siddontang** [comment]
LGTM 👍

**Comment by BusyJay** [comment]
LGTM

**Comment by frewsxcv** [comment]
Version 1.1.0 has been released which includes these changes.

**Comment by BusyJay** [comment]
Thx!

**Pull request merged**